File: QE-selenium/python/Activity10.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.common.action_chains import ActionChains
from selenium.webdriver.common.keys import Keys



with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/drag-drop")
    print(driver.title)
    ball = driver.find_element(By.XPATH,"//img[@id='ball']")
    dropzone1 = driver.find_element(By.XPATH,"//div[@id='dropzone1']")
    dropzone2 = driver.find_element(By.XPATH,"//div[@id='dropzone2']")
    ActionChains(driver).drag_and_drop(ball, dropzone1).perform()
    if driver.find_element(By.XPATH,"//div[@id='dropzone1']/span").text == "Dropped!":
        print("Dropped in Dropzone-1")
    ActionChains(driver).drag_and_drop(ball, dropzone2).perform()
    if driver.find_element(By.XPATH,"//div[@id='dropzone2']/span").text == "Dropped!":
        print("Dropped in Dropzone-2")
    driver.quit()

File: QE-selenium/java/Activity10.java
package demos;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Actions;

public class Activity10 {
    public static void main(String[] args) {

        WebDriver driver = new FirefoxDriver();
        driver.get("https://training-support.net/webelements/drag-drop");
        System.out.println(driver.getTitle());

        Actions actions = new Actions(driver);

        WebElement ball = driver.findElement(By.xpath("//img[@id='ball']"));
        WebElement dropzone1 = driver.findElement(By.xpath("//div[@id='dropzone1']"));
        WebElement dropzone2 = driver.findElement(By.xpath("//div[@id='dropzone2']"));

        actions.dragAndDrop(ball, dropzone1).perform();

        if (driver.findElement(By.xpath("//div[@id='dropzone1']/span")).getText().equals("Dropped!")) {
            System.out.println("Dropped in Dropzone-1");
        }

        actions.dragAndDrop(ball, dropzone2).perform();

        if (driver.findElement(By.xpath("//div[@id='dropzone2']/span")).getText().equals("Dropped!")) {
            System.out.println("Dropped in Dropzone-2");
        }

        driver.quit();
    }
}
